src/webchat-ui/components/presentational/Button.tsx

```diff
@@ -12,46 +13,52 @@ export default styled.button<IColorProps>(({ color, theme }) => {
     }
 
     if (color === 'primary') {
```

Comment:
Here I don't get why we set secondaryColor if when color is equal to 'primary'

Reply:
According to the design specs, the so called "primary button" is using the so called "secondary colors"